api/internal/core/store/mock_interface.go

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package store
+
+import (
+	"context"
+
+	"github.com/stretchr/testify/mock"
+)
+
+type MockInterface struct {
+	mock.Mock
+}
+
+func (m *MockInterface) Get(key string) (interface{}, error) {
+	ret := m.Called(key)
+
+	if rf, ok := ret.Get(0).(func(key string) (interface{}, error)); ok {
```

Comment:
Could we just use a type assert switch statement here? The type assertions are all around.

Reply:
Not very agree. Because this file is generated by mockery, and I do little modification.
So maybe we should keep the original structure.

Reply:
Better to add some comments to mark that the file is generated automically, just like kubebuilder. Otherwise we cannot distinguish the code source.